doc/news/_preparation_next_release.md

```diff
@@ -154,6 +154,12 @@ you up to date with the multi-language support provided by Elektra.
     - `ReadableKey` now implements `equals` and `hashCode` in line with the contract for `int Key::compareTo(Key)`
   - `ReadableKey`/`Key` no longer implements `Iterable<String>` for iterating over the parts of a key's name - use `Iterator<String> ReadableKey::keyNameIterator ()` instead
   - `Key` now implements `Iterable<Key>` to iterate over a key's meta data `ReadableKey`s
+  - Fixed API method typo: Renamed `ReadableKey::isDirectBelow`/`Key::isDirectBelow` to `isDirectlyBelow`
+- `KeyNameIterator` and `KeySetIterator` are now package private
+- `KeySetAppendException` has been renamed to `KeySetException` and now conveys general `KeySet` related exceptional states
+- `KeySet` now implements `SortedSet`
```

Comment:
This is an extremely short description: No link to your source nor to SortedSet API docu. It does not say which benefits there are for the user. An updated tutorial would also be useful.

Reply:
I've updated the release notes